pkg/apis/autoscaling/v1/fleetautoscaler_test.go

```diff
@@ -114,6 +115,11 @@ func TestFleetAutoscalerValidateUpdate(t *testing.T) {
 	})
 
 	t.Run("bad sync interval seconds", func(t *testing.T) {
+		if !runtime.FeatureEnabled(runtime.FeatureCustomFasSyncInterval) {
+			// Do not run test if FeatureCustomFasSyncInterval is not enabled
+			return
```

Comment:
Rather than `return`, let's do a:
```suggestion
		if !runtime.FeatureEnabled(runtime.FeatureCustomFasSyncInterval) {
			// Do not run test if FeatureCustomFasSyncInterval is not enabled
			t.Skip()
```

So we can see that it was skipped 👍🏻

Reply:
Thank you for your review.
I fixed it.